src/main/java/dev/mcnees/moneymapper/MoneyMapperRunner.java

```diff
@@ -0,0 +1,97 @@
+package dev.mcnees.moneymapper;
+
+import java.io.File;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.sql.DataSource;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobParameter;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersBuilder;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MoneyMapperRunner implements ApplicationRunner {
+
+	private static final Logger log = LoggerFactory.getLogger(MoneyMapperRunner.class);
+
+	private final JobLauncher jobLauncher;
+
+	private final Job moneyMapperJob;
+
+	private final JdbcTemplate jdbcTemplate;
+
+	@Value("${qfx_directory:qfx_files}")
+	private String qfx_directory_parameter;
+
+	@Value("${output_directory:output}")
+	private String output_directory_parameter;
+
+	public MoneyMapperRunner(JobLauncher jobLauncher, Job moneyMapperJob, DataSource dataSource) {
+		this.jobLauncher = jobLauncher;
+		this.moneyMapperJob = moneyMapperJob;
+		jdbcTemplate = new JdbcTemplate(dataSource);
+	}
+
+	@Override
+	public void run(ApplicationArguments args) throws Exception {
+
+		clearProcessingData();
```

Comment:
I think you're right.  This was leftover code from when I was using a postgres db and didn't want it cleared after each run so I could get a look at the data mid process.  It shouldn't be needed since I switched to H2.
I have a schema.sql that runs automatically now.  I added a `drop if exists` to that file and removed the Java code to clear it.